cli/cmd/doc_test.go

```diff
@@ -0,0 +1,40 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+)
+
+var docFiles = []string{
+	"cloudquery.md",
+	"cloudquery_completion.md",
+	"cloudquery_completion_bash.md",
+	"cloudquery_completion_fish.md",
+	"cloudquery_completion_powershell.md",
+	"cloudquery_completion_zsh.md",
+	"cloudquery_generate.md",
+	"cloudquery_sync.md",
+}
+
+func TestDoc(t *testing.T) {
+	tmpdir, tmpErr := os.MkdirTemp("", "docs_*")
+	if tmpErr != nil {
+		t.Fatalf("failed to create temporary directory: %v", tmpErr)
+	}
+	defer os.RemoveAll(tmpdir)
+	cmd := newCmdRoot()
+	cmd.SetArgs([]string{"doc", tmpdir})
```

Comment:
I didn't know about the `doc` command, should it be listed in the list of available commands?

```
Usage:
  cloudquery [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  generate    Generate initial config file for source and destination plugins
  sync        Sync resources from configured source plugins to destination
```

Reply:
I think it shouldn't because it's mainly for our website. I don't see a user using this

Reply:
Yeah me neither... but in that case we should probably find a better way of generating the files for the website, no need to ship it with our cli.

Reply:
(not a priority, just a side comment)